modules/product/src/Tests/ProductAdminTest.php

```diff
@@ -50,11 +50,13 @@ function testAddCommerceProductAdmin() {
    * Tests creating a product with an existing SKU.
    */
   function testAddCommerceProductExistingSkuAdmin() {
-    $product = $this->createEntity('commerce_product', array(
+    $product = $this->createEntity(
+      'commerce_product', array(
       'sku' => $this->randomMachineName(),
       'title' => $this->randomMachineName(),
       'type' => 'product'
-    ));
+    )
```

Comment:
The formatting is a little wonky for this entire file, will reformat this at the same time.